### ISSUE TITLE
Apps freeze rM2 since 2.11 update

### ISSUE THREAD

**Issue by RobFreundlich** [rmkit-dev/rmkit#161]
I got the 2.11 update a few days ago. Since then, if I try to run iago or remux, my rM2 freezes and I have to ssh into it and do a reboot. Other commands run via genie (for example, a simple `echo test > /home/root/foo.txt` ) work fine, so I suspect it's something to do with `lamp` or whatever is used to interact with the display.

**Comment by Eeems** [comment]
What version of rm2fb do you have installed?

**Comment by raisjn** [comment]
yes, eeems is correct. need rm2fb 0.12 which should hit stable this week. it's currently in testing. you likely have 0.10

**Comment by raisjn** [comment]
this is in stable, closing out

**Comment by RobFreundlich** [comment]
I'll grab it tomorrow and try it out. Thanks!

**Comment by RobFreundlich** [comment]
I just updated to rm2fb-client version 1:0.0.12-1, reinstalled/enabled iago and remux, and tried them. I have the same problem as before. Here's the package info on the three packages:

```
reMarkable: ~/ opkg info iago
Package: iago
Version: 0.1.0-4
Depends: rm2fb-client, lamp, display
Status: install user installed
Section: utils
Architecture: rmall
Maintainer: raisjn <of.raisjn@gmail.com>
Size: 125713
Filename: iago_0.1.0-4_rmall.ipk
Description: overlay for drawing shapes via stroke injection
Installed-Time: 1640638227

reMarkable: ~/ opkg info remux
Package: remux
Version: 0.1.13-1
Depends: display, rm2fb-client
Status: install user installed
Section: launchers
Architecture: rmall
Maintainer: raisjn <of.raisjn@gmail.com>
Size: 172784
Filename: remux_0.1.13-1_rmall.ipk
Description: Launcher that supports multi-tasking applications
Installed-Time: 1640638589

reMarkable: ~/ opkg info rm2fb-client
Package: rm2fb-client
Version: 1:0.0.12-1
Depends: display
Replaces: rm2fb
Conflicts: rm2fb
Status: install user installed
Section: devel
Architecture: rm2
Maintainer: raisjn <of.raisjn@gmail.com>
Size: 916230
Filename: rm2fb-client_1_0.0.12-1_rm2.ipk
Description: Shim to make reMarkable 1 apps work on reMarkable 2
Installed-Time: 1640638210

reMarkable: ~/
```

**Comment by raisjn** [comment]
`display` also needs to be updated to 0.0.12

**Comment by RobFreundlich** [comment]
Updating `display` helped a lot. But I still get occasional freezes that require a restart.

**Comment by raisjn** [comment]
what applications freeze for you? is it typically after suspend/resume cycle or while using iago?

**Comment by RobFreundlich** [comment]
I'm seeing it in iago and remux.

**Comment by RobFreundlich** [comment]
Well, drat. It looks like the `display` package is incompatible with the screen sharing function in the desktop application. I did a screenshare, and was able to get a static image of my document as it appeared at the time I started the share, but I couldn't get any live updates as I turned pages or write/drew on the page. I started pulling add-ons off of my tablet one by one until the share worked correctly, and it turned out to be `display` that was the culprit. Since it's at the bottom of all of these apps, I'm kind of between a rock and a hard place - do I use these apps, or do I get the full benefit of screen share?

Sigh.
